### ISSUE TITLE
name 'prettydate' is not defined

### ISSUE THREAD

**Issue by ilvalle** [niphlod/cs_monitor_plugin#11]
I've just got this ticket:

```
Traceback (most recent call last):
  File "/home/www-data/web2py/gluon/restricted.py", line 220, in restricted
    exec ccode in environment
  File "/home/www-data/web2py.git/applications/vtraffic/views/plugin_cs_monitor/analyze_task.html", line 125, in <module>
    series: series
NameError: name 'prettydate' is not defined
```

**Comment by niphlod** [comment]
prettydate should be globally available in any app.
https://github.com/web2py/web2py/blob/master/applications/welcome/models/db.py#L43

**Comment by ilvalle** [comment]
It is available in the welcome app. I removed those lines unneeded on my
app, https://github.com/ilvalle/vtraffic/blob/master/models/db.py
I will add them back.
Even if I am not sure the plugin should relies on the fact that it is
always globally available, isn't it?
If you depends on prettydate, my suggestion is to check if it is defined,
otherwise import it explicitly

 Paolo

2014-05-02 17:26 GMT+02:00 niphlod notifications@github.com:

**Comment by niphlod** [comment]
never though it won't be available, but it's an easy fix ^_^

**Comment by niphlod** [comment]
closed by e516286
